exporter/signalfxexporter/README.md

```diff
@@ -26,6 +26,7 @@ be used instead. If path is not specified, `/v2/datapoint` is used.
 If `realm` is set, this option is derived and will be `https://api.{realm}.signalfx.com/`. If a value is explicitly
 set, the value of `realm` will not be used in determining `api_url`. The explicit value will be used instead.
 - `log_dimension_updates` (default = `false`): Whether or not to log dimension updates.
+- `access_token_passthrough`: Whether to use `"com.splunk.signalfx.access_token"` metric resource label, if any, as SFx access token.  In either case this label will be dropped during final translation.  Intended to be used in tandem with identical configuration option for [SignalFx receiver](../../receiver/signalfxreceiver/README.md) to preserve datapoint origin.
```

Comment:
Since it's new behavior and can override the specified access token it seems prudent to flag it wherever applicable to me.  The downside of it being the default is negligible so I've updated the default config in the exporter but it will still default to false in the receiver.  This is largely a product of it being disabled by default in the SFx gateway where the feature originates.